cmd/notation/cert.go

```diff
@@ -184,6 +247,23 @@ func printCertificateSet(s config.CertificateMap) {
 	for _, ref := range s {
 		fmt.Printf(format, ref.Name, ref.Path)
 	}
+
+	var maxKeyIDSize int
+	for _, ref := range k {
+		if len(ref.Name) > maxNameSize {
+			maxNameSize = len(ref.Name)
+		}
+		if len(ref.ID) > maxKeyIDSize {
+			maxKeyIDSize = len(ref.ID)
+		}
+	}
```

Comment:
Why are we doing this? something peculiar to golang's `sprintf`(line 262)? Can we remove this piece of code?